### PR TITLE
Rename @privateRemarks TSDoc tag to @hidden

### PR DESCRIPTION
**Pull request by erinha**
Turns out that the @privateRemarks tag is not currently supported by TypeDoc generator used by MS Docs for TypeScript documentation. To suppress comments from publish on docs.microsoft.com, the recommendation is to use @hidden.

### CODE REVIEW

File: packages/teams-js/src/private/interfaces.ts [hunk 1]
```diff
@@ -56,108 +56,108 @@ export enum ViewerActionTypes {
 }
 
 /**
- * * @privateRemarks
+ * * @hidden
```
Comment (erikadoyle) [inline packages/teams-js/src/private/interfaces.ts:59]: There's an extraneous asterisk before @hidden (line 59) that's causing the "Hide from docs" text to generate documentation

**Review by erikadoyle — COMMENTED** [review]
Just that one issue--otherwise the docs generate great when I run typedoc locally.

File: packages/teams-js/src/private/interfaces.ts (hunk 1 above)
Comment (erinha) [reply packages/teams-js/src/private/interfaces.ts:59]: Great catch!

**Comment by aeunms** [comment]
Looks like there's some @private tags in pages.ts that didn't get updated to @privateRemarks before, I think those need to be @hidden as well.

**Review by aeunms — APPROVED** [review]
LGTM!

**Pull request merged**